61.md

```diff
@@ -0,0 +1,64 @@
+NIP-61
+======
+
+Event Sets
+----------
+
+`draft` `optional`
+
+An event set is a group of events with the same pubkey and `n` (set "n"ame) tag.
+An event can have many `n` tags, thus being member of many event sets.
+
+To add an event to the "Close Friends" set,
+add a "close-friend" (prefer lowercase English words in singular form with dash separator) `n` tag to the event.
+
+To remove an event from a "Close Friends" set,
+remove the "close-friend" `n` tag from the event or send a deletion event (`kind:5`).
+
+An event set itself is referenced by an `s` tag
+or by an `nset` [NIP-19](19.md) entity using [NIP-21](21.md) URI.
+
+## Referencing
+
+An event set is referenced by an `s` tag.
+It references a set of events from an author instead of a single one:
+
+`["s", "<kind, optional>:<32-bytes lowercase hex of a pubkey>:<n tag value>", "<recommended relay URL, optional>"]`
```

Comment:
I think we should use some other format here, just to be different from normal replaceable event references.

```suggestion
`["s", "<n-tag value>#<32-bytes lowercase hex of a pubkey>", "<recommended relay URL, optional>"]`
```

And why is the kind optional? It shouldn't be optional, that's bad idea. We should either always omit it or always include it.

Maybe we come up with a reference format for including multiple kinds here if that's a requirement.

Reply:
The n-tag should be last cause it can contain any char, including the ones used as separator.
Empty kinds may be useful when event set may contain events from too many kinds or even from kinds that were not speced?
Not sure if it is good to introduce another separator considering that the tag name dictates the value format.
Added comma-separated kinds.

Reply:
Ended up with 1 required kind to lower complexity and cause it aligns with recent changes regarding "Set Item Reference" events.